code/modules/cargo/packs_recreation.dm

```diff
@@ -129,4 +129,14 @@
 					/obj/item/toy/weapon/sword)
 	cost = 150
 	crate_name = "toy weapons crate"
+	group = "Recreation"
+
+/datum/supply_pack/oddities
+	name = "Luxury Imports Crate"
+	contains = list(/obj/random/lonestar_oddities,
+					/obj/random/lonestar_oddities,
+					/obj/random/lonestar_oddities,
+					/obj/random/lonestar_oddities)
+	cost = 1200
```

Comment:
```suggestion
	cost = 1980
```

Reply:
A bit much, I think. At 1200 a crate, that's 300 an oddity. These oddities are all pretty weak and even the ones that have a gimmick (like the Kriosan officer sword), don't really have a gimmick that skyrockets them to super useful. At 1980 (har, har), that's roughly 500 an oddity, so 600 for any noticeable return on sale. Seeing as this is meant to add traffic to Lonestar and further cement them as the bargain bin version of every other department, not sure increasing the cost is worth it. _Not to mention I have a handful of oddities left that aren't useful at all and will be further weighing the pool towards weakness._